TESTING.md

```diff
@@ -381,6 +381,26 @@ cargo make -e TESTSYS_TEST=migration test
 
 To see the state of the tests as they run use `cargo make watch-test`.
 
+### Testing Workloads
+
+Workload tests are tests designed to run on a pod in a cluster.
```

Comment:
We can do ECS workload tests as well, right?

```suggestion
Workload tests are tests designed to run as an orchestrated container.
```

Reply:
We can't do ECS yet. That will come in the next TestSys release.